src/Entities/AvsResponse.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rebilly\Entities;
+
+use Rebilly\Rest\Entity;
+
+/**
+ * Class Gateway
+ */
+final class AvsResponse extends Entity
```

Comment:
It's not an entity, this object does not have `id`. You should use `Resource` in such classes.